src/pudl/transform/nrelatb.py

```diff
@@ -289,7 +295,18 @@ def _core_nrelatb__transform_start(raw_nrelatb__data):
         # we are dropping records that are completely dupes - not just dupes based on IDX_ALL
         .drop_duplicates(keep="first")
     )
-    assert not any(nrelatb.duplicated(IDX_ALL))
+    # In 2024, we see many records which are completely identical except for their core
+    # metric key, which we treat as a bad primary key. We should drop these duplicates.
+    logger.info(
```

Comment:
What is a core metric key and why is it a bad primary key?

Reply:
expanded and referenced issues in the comments